main.py

```diff
@@ -25,6 +25,21 @@ async def on_ready() -> None:
 
 
 # Commands
+
+
+# TODO: Modify the help command to provide descriptions and category title
```

Comment:
Other than this TODO, lgtm.

Reply:
Yeah that was more of a reminder for me and anyone else pulling at the time!